### DISCUSSION TITLE
Mark synced in local DB

### DISCUSSION THREAD

**Discussion by percramer** [Mimetis/Dotmim.Sync#889] [Q&A]
Hi,

so I am doing some testing for a setup if mine. I have got an application that works online / offline. So there is a local sqlite db and a remote SQL server. The application gets some new values every 15 seconds and it then writes that to the sqlite db and to the SQL server directly. I now would like to start using this library for when the connection with the SQL server goes down. I can then keep storing the data locally and just sync when the connection is back. That is pretty straightforward with this library.

But when there is a connection with the SQL server I would still like to write the data directly to the server. Is there a way how I can mark the local data (in the sqlite databas)that was successfully written to the SQL server as "already synced / present"?

**Comment by Mimetis** [comment]
You can use the timestamp stored in the `scope_info_client` table
Have a look on the `_get_changes` stored procedure to see how DMS is getting rows "not yet synced"

Please if your question is not an Issue, plz consider creating a thread in the discussion's areas instead

**Comment by percramer** [comment]
ok, sorry I didn't notice the discussion option. I will use that the next time (and for now take a closer look at the SP)
